### ISSUE TITLE
German - English 2019-01 Wikipedias

### ISSUE THREAD

**Issue by Ope30** [gnosygnu/xowa#317]
Before I even say anything, I'm wishing everyone here in the XOWA forum a great new year! Let's see what this year has to offer, particularly regarding XOWA.

What really interests me are the things you and desb42 have been working on intensively. Will there be a huge update for XOWA in 2019? That would be fantastic! I really admire the HTTP-Server feature, just so you know.

The latest builds (DE-EN) are from 2018-08.  Would it be possible for you uploading two new ones? Or should I ask you next month/february? Your house, your rules.

**Comment by gnosygnu** [comment]
Hey, thanks for the reminder. I'm really trying to get a new build out soon. I made an issue to track blocking items to the next build in #310 .

It's hard to say when it will be, but it's definitely a priority . Please feel free to bump this issue again as time passes.

Thanks!

**Comment by Ope30** [comment]
> Hey, thanks for the reminder. I'm really trying to get a new build out soon. I made an issue to track blocking items to the next build in #310 .

> It's hard to say when it will be, but it's definitely a priority . Please feel free to bump this issue again as time passes.


I would definitely try the new build, no doubt about it. Cool. :)


Yup. I will bump this up in february, probably the first day of the month (just incase). If you want I can come back in march instead, as you please.

**Comment by Ope30** [comment]
Heya!

Just wanted to bump this up. Is 2019-02 DE/EN possible?

**Comment by gnosygnu** [comment]
Yeah, sorry. Been busy between late work hours and the various issues here.

I'm about 90% done with the new html database format (the one without hzip). I'm going to try to finish up the code by next weekend. If that fails, then I'll hold my breath and try the old hzip one and hope the issues desb42 found are not show-stoppers

So yeah, Feb is my new target date. If it means anything, this has been a major item on my side. I just haven't had the time to work on it.

Thanks.

**Comment by Ope30** [comment]
> Yeah, sorry. Been busy between late work hours and the various issues here.

> I just haven't had the time to work on it.

No worries!

**Comment by Ope30** [comment]
Heyyo.



How's it going so far, how's XOWAs development!? Will there be a new build (DE-EN) this rest of the month? :D

**Comment by gnosygnu** [comment]
Sorry. Been sick since Tuesday and still recovering. I think I have a good 2019-01 German Wikipedia. I was running a 2019-03 one, but ran into a bunch of errors which will require a little code-patching. I'm going to take another look at both tomorrow, and try to upload one of them during the week.

**Comment by Ope30** [comment]
> Sorry. Been sick since Tuesday and still recovering. I think I have a good 2019-01 German Wikipedia. I was running a 2019-03 one, but ran into a bunch of errors which will require a little code-patching. I'm going to take another look at both tomorrow, and try to upload one of them during the week.


Hey.



I'm sorry to hear about that. I hope you get better soon! 👍 

Also, you shouldn't be the one apologizing, instead I should. I really wish I knew how to do this all by myself so that I do not have to get on your nerves most of the time (that's how I feel like before asking). I don't know the way you build the wikis, but I believe it all depends on your computer speed at the end. I have a good PC, so that shouldn't be a problem for me. Downloading doesn't take long for me either. If only I knew how to do it alone, I think I would spare you alot of time. :)
We can discuss this later on in the future (once you're fine and in the mood of course). You could teach me your way as easy as possible.

Regarding 2019-01 German Wikipedia, that sounds promising! Thank you for that. Are you including 2019-01 EN as well? Just asking. Remember, no time pressure.
